ikarus/finiteelements/mechanics/kirchhoffloveshell.hh

```diff
@@ -189,14 +189,25 @@ public:
    * \param local Local position vector.
    * \return calculated result
    *
-   * \tparam resType The type representing the requested result.
+   * \tparam RT The type representing the requested result.
    */
-  template <ResultType resType>
+  template <template <typename, int, int> class RT>
```

Comment:
We can also disable this function completly if the result type is not avaibable. Basically in all element add `requires canProvideResultType<RT>`

Reply:
Then the static assert is not needed in the function

Reply:
Here i don't see a static_Assert only the throw. and yes this would not be needed.